lambdas/layers/umccr_utils/umccr_utils_test/sample-1

```diff
@@ -0,0 +1,49 @@
+[Header],,,,,,,,,,
```

Comment:
Please mock, obfuscate or randomise this? We are in public repo. So, discourage using any actual content for unit test purpose. Please do force commit override this so that it can go away from history all together.

Reply:
SampleSheets normally don't contain identifiable information, but it's a good strategy in general to randomise/obfuscate real data.